storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -170,7 +170,23 @@ namespace {
 std::vector<bool> GetAttrsUseIndicator(TABLE *table) {
   int col_id = 0;
   std::vector<bool> attr_uses;
+  enum_sql_command sqlCommand = SQLCOM_END;
+  if (table->in_use && table->in_use->lex)
+    sqlCommand = table->in_use->lex->sql_command;
```

Comment:
1. Uniform var names, refs: 
https://google.github.io/styleguide/cppguide.html#Variable_Names

2. Wrap the expression with brackets